code/modules/mob/living/carbon/xenomorph/life.dm

```diff
@@ -44,6 +44,8 @@
 /mob/living/carbon/xenomorph/handle_fire()
 	. = ..()
 	if(.)
+		if(resting && fire_stacks > 0)
+			adjust_fire_stacks(-1)	//Passively lose firestacks when not on fire while resting and having firestacks built up.
```

Comment:
Resting and maybe resisting as well? This PR made me realize that xeno firestack icon isnt actually bugged, it displays the real remaining firestacks on a xeno and the default method to remove fire is resisting.

Resting runs counter to the standard xeno method of removing fire and will need to be displayed in chat or in a balloon_alert maybe?

```suggestion
			adjust_fire_stacks(-1)	//Passively lose firestacks when not on fire while resting and having firestacks built up.
			balloon_alert(src, "Firestack reduced")
```

I dont know if src is right though.

Reply:
Well, this is for the passive decay of "status buildup" that the next "status trigger" aka ignition would use to have stronger effects. I don't think it needs to be displayed more than the icon decreasing as it'd just spam you with messages while resting. Besides, it *should* be pretty rare to occur now that jelly isn't comedically backfiring like right now (noone ever noticed that though)

Reply:
```suggestion
			adjust_fire_stacks(-1)	//Passively lose firestacks when not on fire while resting and having firestacks built up.
			to_chat(src, span_bold("Firestack reduced due to resting."))
```

Possibly instead of a balloon alert then just to_chat? Passive actions even with the flame hud icon changes seem iffy. I know sunder has no chat response but at least it has a game tab value that increases to signify something is happening.

Fancy chat will collapse the to_chat spam to just (x3) and so on so there's no flooding of the chat box.

Reply:
Nothing else that has passive effects gives you messages for what it does and this is a super-edge-case thing anyways since most times if you get stacks you are getting set on fire and being extinguished tends to happen at 0 stacks.